server/edusync/src/main/java/com/codestates/edusync/study/studygroup/dto/StudygroupResponseDto.java

```diff
@@ -43,14 +43,29 @@ public static class StudyLeader {
         private String nickName;
     }
 
+    /**
+     * 스터디 조회
+     */
     @NoArgsConstructor
     @Getter
     @Setter
     public static class DtoList {
         private Long id;
         private String title;
-        private List<SearchTag> tags;
+        //private
     }
+
+    /**
+     * 스터디 조회 시, 태그
+     */
+    @NoArgsConstructor
+    @Getter
+    @Setter
+    public static class tagDto {
+        private String key;
+        private String value;
+    }
```

Comment:
클래스 이름의 첫글자가 대문자였으면 좋겠습니다.

```suggestion
    @NoArgsConstructor
    @Getter
    @Setter
    public static class TagDto {
        private String key;
        private String value;
    }
```